client/src/components/SignIn/SignIn.jsx

```diff
@@ -1,7 +1,158 @@
+        value: email,
+        message: "Email is required",
+        errorMessage: "Please enter a valid email",
+      },
+      {
+        name: "password",
+        value: password,
+        message: "Password is required",
+        minLength: 8,
+        errorMessage: "Password must be at least 8 characters long",
+      },
+    ];
+
+    let allValid = true;
+
+    fields.forEach(({ name, value, message, errorMessage, minLength }) => {
+      if (value.trim() === "") {
+        setError(name, message);
+        allValid = false;
+      } else if (minLength && value.length < minLength) {
+        setError(name, errorMessage);
+        allValid = false;
+      } else {
+        setSuccess(name);
+      }
+    });
+
+    return allValid;
+  };
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    const signInData = {
+      email: signInValues.email,
+      password: signInValues.password,
+    };
+
+    if (validateInputs() === true) {
+      try {
+        const result = await handleSignIn({ signInData });
+
+        if (result.success) {
+          window.location.href = `/profile/${result.id}`;
+        } else {
+          throw new Error(result.error);
+        }
+      } catch (error) {
+        if (error.response === true && error.response.status === 422) {
+          setError("form", "Incorrect email or password");
+        } else {
+          setError("form", "Incorrect email or password");
+        }
+        console.error("Sign-in error:", error);
+      }
+    }
+  };
+
   return (
-    <div>
-      <h1>Sign in</h1>
-    </div>
+    <Form className="logInComponent" method="post" onSubmit={handleSubmit}>
+      <img src={logo} alt="" className="imageLogo" />
+      <div className="logIn">
+        <p>Acces to your profile</p>
+        <label>
+          <input
+            className="emailInput container"
+            type="email"
+            id="email"
+            name="email"
+            placeholder="Email"
+            value={signInValues.email}
+            onChange={handleChange}
+            autoComplete="off"
+            required
+          />
+          {signInErrors.email !== "" && (
+            <span className="error">{signInErrors.email}</span>
+          )}
+        </label>
+        <label>
+          <input
+            className="passwordInput container"
+            type="password"
+            id="password"
+            name="password"
+            placeholder="Password"
+            value={signInValues.password}
+            onChange={handleChange}
+            autoComplete="off"
+            required
+          />
+          {signInErrors.password !== "" && (
+            <span className="error">{signInErrors.password}</span>
+          )}
+        </label>
+        {signInErrors.form !== "" && (
+          <span className="error">{signInErrors.form}</span>
+        )}
+        <a href="/signin" className="forgetPassword">
+          <p>
+            <span>Forget password?</span>
```

Comment:
The `<p>` tag is a block-level element and should typically be used to contain paragraphs of text, not as a wrapper for other elements. (remove it)

You could use the `Link `tag since it's React.